display/ssd1306.h

```diff
@@ -274,10 +303,31 @@ class SSD1306 : public I2CDevice, Looper, StateMachine, SaberBase, private Audio
   }
 
   void SB_NewFont() override {
-    if (IMG_font) {
-      // Overrides message from below..
-      ShowFile(&IMG_font);
-    }
+    ShowFile(&IMG_font);
+  }
+
+  void SB_On() override {
+    ShowFile(&IMG_on);
+    imgon_ = true;
```

Comment:
If we really just need to know if it is IMG_on or not, we can just check effect == &IMG_on in ShowFile instead.
If there are potentially other effects that needs this to be true, then it should probably be an argument to ShowFile()

Reply:
fixed.  I don't see the need for other effects, so just setting the bool in showfile